WalletWasabi/Wallets/Wallet.cs

```diff
@@ -80,7 +81,7 @@ private set
 	public string WalletName => KeyManager.WalletName;
 
 	/// <summary>Unspent Transaction Outputs</summary>
-	public ICoinsView Coins { get; private set; }
+	public CoinsRegistry Coins { get; private set; }
```

Comment:
Why? 🤔

Reply:
I think this is correct. Also, @kiminuo did the same in his PR I think.